misc/discovery_schema.json

```diff
@@ -393,7 +393,7 @@
                             },
                             "num_oid": {
                                 "type": "string",
-                                "pattern": "^(\\.\\d+)+\\.?\\{\\{ \\$index \\}\\}(\\.\\d+)*$"
```

Comment:
This seems to be a necessary step in order to poll sensors using YAML if they don't come in table format.